python/paddle/fluid/layers/layer_function_generator.py

```diff
@@ -205,6 +208,29 @@ def func(*args, **kwargs):
     return func
 
 
+def generate_layer_fn_noattr(op_type):
```

Comment:
Have you printed the attrs fo these operators and double check that those attrs are not needed?

Reply:
I have checkout the cc files here
https://github.com/PaddlePaddle/Paddle/blob/f7af695801dc17dfd3e03c73b8af20ebdbc1a6e4/paddle/fluid/operators/activation_op.cc#L455-L472

Only mkldnn attr maybe probably needed, but since mkldnn is not actually used or solid enough, so it should not be exposed right now, we can add it when mkldnn is ready for all these ops.